### PR TITLE
WIP: docs: Module 5 Starport - code sample integration 1 of 3 (B9lab updates 4-3)

### PR DESCRIPTION
**Pull request by coldice**
Docs content update for Module 5, Section _Starport_ - part of the [b9lab-content](https://github.com/cosmos/sdk-tutorials/tree/master/b9lab-content).

This PR adds code samples to the content. The code shown here is taken from the [code example repo](https://github.com/cosmos/b9-checkers-academy-draft/) and has been hardly adjusted. The repo has been reviewed so no need to review it here. What we are looking for is to make sure that the introductory texts are not misleading or even downright wrong.

The Starport code sample integration PRs are split into three parts, also see #852 and #854

### Content state

**✔️ Completed**

### Technical review
- [x] requested
- [ ] completed

### Language review
- [ ] requested
- [ ] completed

### CODE REVIEW

File: b9lab-content/5-my-own-chain/03-starport-03-stored-game.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
+---
+title: The Stored Game Object
+order: 4
```
Comment (toschdev) [inline b9lab-content/5-my-own-chain/03-starport-03-stored-game.md:3]: Are there a few articles before the tutorial? 

There might be missing the 

`starport scaffold chain github.com/username/checkers`

command to get started, maybe some other commands that I am missing before this chapter?

File: b9lab-content/5-my-own-chain/03-starport-03-stored-game.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+const (
+    StoredGameKey = "StoredGame-value-"
+)
+```
+Which will be used to prefix the keys at which the objects are stored.
+
+## Protobuf Objects
+
+As you know, Starport creates the Protobuf objects first before compiling them. Therefore you have the `NextGame` object:
```
Comment (toschdev) [inline b9lab-content/5-my-own-chain/03-starport-03-stored-game.md:55]:
```suggestion
As you know, Starport creates the Protobuf objects first in the `proto` directory before compiling them. Therefore you have the `NextGame` object:
```

File: b9lab-content/5-my-own-chain/03-starport-03-stored-game.md (hunk 1 above)
Comment (xavierlepretre) [reply b9lab-content/5-my-own-chain/03-starport-03-stored-game.md:3]: Added a couple paragraphs as a glue.

Comment (CitMC) [reply b9lab-content/5-my-own-chain/03-starport-03-stored-game.md:3]: @xavierlepretre is this feedback implemented completely? If yes, please resolve this conversation.

**Comment by coldice** [comment]
Merged in b9lab-content-updates-rc1 (no diff to this branch)

**Pull request closed without merging**